fhir-info-gateway/keycloak-config.json

```diff
@@ -0,0 +1,1209 @@
+            "attributes": {
+              "consent.screen.text": "Permission to retrieve current logged-in user"
+            },
+            "mappers": {
+              "fhirUser Mapper": {
+                "protocol": "openid-connect",
+                "protocolmapper": "oidc-patient-prefix-usermodel-attribute-mapper",
+                "config": {
+                  "user.attribute": "resourceId",
+                  "claim.name": "fhirUser",
+                  "jsonType.label": "String",
+                  "id.token.claim": "true",
+                  "access.token.claim": "false",
+                  "userinfo.token.claim": "true"
+                }
+              }
+            }
+          },
+          "launch/patient": {
+            "protocol": "openid-connect",
+            "description": "Used by clients to request a patient-scoped access token",
+            "attributes": {
+              "display.on.consent.screen": "false"
+            },
+            "mappers": {
+              "Patient ID Claim Mapper": {
+                "protocol": "openid-connect",
+                "protocolmapper": "oidc-usermodel-attribute-mapper",
+                "config": {
+                  "user.attribute": "resourceId",
+                  "claim.name": "patient_id",
+                  "jsonType.label": "String",
+                  "id.token.claim": "false",
+                  "access.token.claim": "true",
+                  "userinfo.token.claim": "false"
+                }
+              },
+              "Patient ID Token Mapper": {
+                "protocol": "openid-connect",
+                "protocolmapper": "oidc-usersessionmodel-note-mapper",
+                "config": {
+                  "user.session.note": "patient_id",
+                  "claim.name": "patient",
+                  "jsonType.label": "String",
+                  "id.token.claim": "false",
+                  "access.token.claim": "false",
+                  "access.tokenResponse.claim": "true"
+                }
+              },
+              "Group Membership Mapper": {
+                "protocol": "openid-connect",
+                "protocolmapper": "oidc-group-membership-mapper",
+                "config": {
+                  "claim.name": "group",
+                  "full.path": "false",
+                  "id.token.claim": "true",
+                  "access.token.claim": "true",
+                  "userinfo.token.claim": "true"
+                }
+              }
+            }
+          },
+          "online_access": {
+            "protocol": "openid-connect",
+            "description": "Request a refresh_token that can be used to obtain a new access token to replace an expired one, and that will be usable for as long as the end-user remains online.",
+            "attributes": {
+              "consent.screen.text": "Retain access while you are online"
+            }
+          },
+          "system/*.rs": {
```

Comment:
I see this one gets loaded but all the other don't seem to get loaded in keycloak. Do you know what is happening to the others?